crates/re_renderer/src/renderer/point_cloud.rs

```diff
@@ -75,12 +75,16 @@ mod gpu_data {
         pub end_padding: [wgpu_buffer_types::PaddingRow; 16 - 1],
     }
 
-    /// Uniform buffer that changes for every batch of points.
+    /// Uniform buffer that changes for every 1 of points.
```

Comment:
weird sentence. Did you mean

```suggestion
    /// Uniform buffer that changes for every point.
```

?

Reply:
![image](https://user-images.githubusercontent.com/1220815/236780205-36fc7339-34d9-497e-965a-7e6379e41963.png)